Mage/src/main/java/mage/game/events/GameEvent.java

```diff
@@ -493,6 +493,7 @@ combines all permanent damage events to a single batch (event) and split it per
         STAY_ATTACHED,
         ADD_COUNTER, COUNTER_ADDED,
         ADD_COUNTERS, COUNTERS_ADDED,
+        REMOVE_COUNTER, REMOVE_COUNTERS,
```

Comment:
Must add comments with description of the params (see another events for example). Also look at the start of `GameEvent` and copy flag description too.